src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -105,6 +105,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       request.getRequestDispatcher("/WEB-INF/view/conversations.jsp").forward(request, response);
       return;
     }
+          
+    if (conversationTitle.length() == 0) {
```

Comment:
Using conversationTitle.isEmpty() is usually preferred for situations like this because it reads a little better. Please replace here and similar situations elsewhere.